lib/modules/manager/terragrunt/extract.spec.ts

```diff
@@ -14,6 +14,20 @@ describe('modules/manager/terragrunt/extract', () => {
       expect(res?.deps.filter((dep) => dep.skipReason)).toHaveLength(5);
     });
 
+    it('extracts terragrunt sources with depth specified after the branch', () => {
+      const res = extractPackageFile(Fixtures?.get('3.hcl'));
+      expect(res).toMatchSnapshot();
```

Comment:
Do not use `toMatchSnapshot` rather use `toEqual`

We have newer code style guides https://github.com/renovatebot/renovate/blob/main/docs/development/best-practices.md#unit-testing